crocket/src/main/java/com/crocket/view/LevelView.java

```diff
@@ -72,11 +75,34 @@ private JLabel makeLabel(DrawableEntity entity, BufferedImage bufferedImage) {
         return label;
     }
     
-    // TODO: FIX THIS!!!!
     private void drawDirectionLine(DrawableEntity entity) {
-     
+        DrawDirectionLine panel = new DrawDirectionLine();
+        panel.setBounds(entity.getxPosition()-panel.getWidth()/2 +10, entity.getyPosition()-panel.getHeight()/2 +10, panel.getWidth(), panel.getHeight());
+        
+        panel.changeAngle(entity.getCosinus(), entity.getSinus());
+        add(panel, JLayeredPane.DEFAULT_LAYER);
+        panel.repaint();
+    }
+
+    private void drawPowerMeter(){
+        pPanel.setBounds((int)screen.getWidth()/100, 90 * (int)screen.getHeight()/100, pPanel.getWidth(), pPanel.getHeight());
```

Comment:
Where do we get these values from. Could they be constants?

Reply:
fixed